lib/Backends/CPU/tests/CPURecommendationSystemTest.cpp

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "tests/unittests/BackendTestUtils.h"
+
+using namespace glow;
+
+std::set<std::string> glow::backendTestBlacklist = {
+    "RecSys_RWQuantized_SLWS_FC_FP16/0",
+    "RecSys_RWQuantized_SLWS_FP16/0",
+    "RecSys_RWQuantizedFP16_SLWS/0",
+    "RecSys_RWQuantizedFP16AccumFP16_SLWS/0",
+    "RecSys_RWQuantizedFP16_SLWS_FP16/0",
+    "RecSys_Partitioned_RWQuantized_SLWS_FC_FP16/0",
+    "RecSys_RWQuantizedFP16AccumFP16_SLWS_FP16/0",
+    "RecSys_Partitioned_RWQuantized_SLWS_FP16/0",
+    "RecSys_Partitioned_RWQuantizedFP16_SLWS/0",
+    "RecSys_Partitioned_RWQuantizedFP16AccumFP16_SLWS/0",
+    "RecSys_Partitioned_RWQuantizedFP16_SLWS_FP16/0",
+    "RecSys_Partitioned_RWQuantizedFP16AccumFP16_SLWS_FP16/0",
+    "RecSys_SLS_Only/0",
```

Comment:
Let me check -- I generated most of the blacklists programmatically using a bash script basically like
```
for test in $(XyzTest --gtest_list_tests); do
  if ! XyzTest --gtest_filter="$test"; then
    echo $test
  fi
done
```
But for RecSys I did some manual editing because it's so long running, and I probably just made a mistake.

Reply:
I wouldn't mind adding CHECK_IF_ENABLED on everything, either, but I don't have a strong opinion.  I've added it in a lot of places but probably not everywhere (like, OperatorTest I think misses some cases that just happened to pass on all backends).

Reply:
I was also contemplating if `CHECK_IF_ENABLED` could be rolled into a special macro, like `TEST_GLOW_BACKEND(suite, test)`, and I think it can be done but since this diff is already a bit thorny I didn't want to push on it too much.